src/styles/stars.css

```diff
@@ -0,0 +1,15 @@
+.stars {
+  display: flex;
+}
+
+.stars__star {
+  width: 16px;
```

Comment:
Consider using SCSS variables for dimensions like `16px` to improve maintainability and consistency across your stylesheets.